eng/Versions.props

```diff
@@ -12,6 +12,12 @@
       This should get updated each time packages are shipped.
     -->
     <BaselineVersionForPackageValidation>8.0.1</BaselineVersionForPackageValidation>
+    <DefaultTargetFramework>net8.0</DefaultTargetFramework>
+    <!-- dotnet 8.0 versions for running tests -->
+    <DotNetRuntimeVersionForTesting>8.0.8</DotNetRuntimeVersionForTesting>
```

Comment:
I'm wondering how we can keep this one up-to-date. One option would be if we use the `Microsoft.NetCore.App.Ref` package and dependency flow for the version as we know that one will push the update to us. Anyway, no need to fix it now but we should consider how to update this (and the sdk below)

Reply:
`dotnet/runtime` is using `Microsoft.DotNet.ApiCompat.Task` for that. Either should work well.